README.russian.md

```diff
@@ -100,3 +104,22 @@ dofile('res/testing.lua')
 * [Windows Bonus - Flashy Icons & Registry Enhancements](https://github.com/tilkinsc/LuaConsole/wiki/Windows-Bonus----Flashy-Icons-&-Registry-Enhancements)  
 * [Linux Bonus - Shebangs & Desktop Files](https://github.com/tilkinsc/LuaConsole/wiki/Linux-Bonus---Shebangs-&-Desktop-Files)
 
+## Contributors ✨
```

Comment:
I can't accept a change on this. This is automatically generated by a bot. Please revert this change.

Reply:
I also don't understand how it can break something, but I'll revert this change too